src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -2,18 +2,27 @@ package racingcar.view
 
 import racingcar.domain.Car
 import racingcar.domain.Cars
+import racingcar.domain.RacingResult
+import racingcar.domain.Winner
 
-fun printResultMessage() = println("실행결과")
-fun printCars(cars: Cars) {
+private const val DISTANCE_SYMBOL = "-"
+
+fun printResult(result: RacingResult) {
+    println("실행결과")
+    result.elements
+        .forEach { printCars(it) }
+}
+
+private fun printCars(cars: Cars) {
     cars.elements
         .forEach { printCar(it) }
     println()
 }
 
 private fun printCar(car: Car) {
-    val value = car.distance.value
-    val stringBuilder = StringBuilder()
-    (0 until value).map { stringBuilder.append("-") }
+    println(car.name.value + " : " + DISTANCE_SYMBOL.repeat(car.distance.value))
```

Comment:
문자열 템플릿을 사용하면 문자열을 연결하지 않아도 되므로 코드가 간결해집니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/basic-types.html#string-templates

```suggestion
    println("${car.name.value} : ${DISTANCE_SYMBOL.repeat(car.distance.value)}")
```